{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/urls.py

```diff
@@ -19,10 +19,13 @@
 from django.urls import include, path, re_path
 from django.views.static import serve
 
+from .views import HealthView
+
 admin.site.site_header = admin.site.site_title = "{{ cookiecutter.project_name }}"
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("backend/health/", HealthView.as_view(), name="health-check"),
```

Comment:
the route path it would be better with cookiecutter reference to backend `service_slug` 

```suggestion
    path("{{ cookiecutter.service_slug }}/health/", HealthView.as_view(), name="health-check"),
```